tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -437,6 +437,12 @@ public:
    /// When writing a variable size array through Snapshot, it is required that the column indicating its size is also
    /// written out and it appears before the array in the columnList.
    ///
+   /// \attention In multi-thread runs (i.e. when EnableImplicitMT() has been called) threads will loop over clusters of
+   /// entries in undefined order, so Snapshot will produce outputs in which (clusters of) entries will be shuffled with
```

Comment:
Middling comment, but this should read, "in **an** undefined order".

Reply:
fair enough, can you please add a github comment with the "insert suggestion" thingy so I can just accept the correction? :smile:

Reply:
```suggestion
   /// entries in an undefined order, so Snapshot will produce outputs in which (clusters of) entries will be shuffled with
```